### PR TITLE
text and style tweaks before review

### PR DESCRIPTION
**Pull request by ellenbechtel**
Changes made:

- reduced font size a little bit to make it less overwhelming, especially on mobile.
- added color overlays to chapter title images to make them more consistent
- Updated diagrams and moved lots of the text into the caption area.  
- Made a new component called "KeyDynamics" to house all the sidebars of key terminology early on, so we don't have to double-mark key terms and their explanations.
- Edited text to fit the edited diagrams
- added sidebar components to SWEanim and STNLmap, but forgot what key terminology we wanted to highlight.
-----------
Description

Testing:
----------------------------
Before making this pull request, I:
- [x] Cleaned the code the way Vue likes it - run 'npm run lint --fix'        
- [ ] Made sure all tests run
- [ ] Ran WAVE plugin 508 compliance tool

I can confirm this has been checked on:
- [x] Chrome
- [ ] Safari
- [ ] Edge
- [ ] Firefox
- [ ] Samsung Internet
- [ ] Internet Explorer 11 (not supported, but still needs at least a working user redirect page)

### CODE REVIEW

File: src/components/DiagramCarouselNormal.vue [hunk 1]
```diff
@@ -18,7 +18,7 @@
             >
             <img 
               id="diagram-normal-winter"
-              v-img="{group: 'diagram-normal', title: 'Caption about winter in a normal snow year'}"
+              v-img="{group: 'diagram-normal', title: 'A Normal Winter: Soil saturation sets the stage for the coming snow season - whether it was wet or dry the year before determines how saturated it is.  Around October, snow begins to fall and SWE begins to accumulate. Especially in high elevations, snowpacks can be several feet thick and store several inches worth of water.'}"
```
Comment (collnell) [inline src/components/DiagramCarouselNormal.vue:21]: Let's use "Typical" instead of "normal"

File: src/components/DiagramsHigh.vue [hunk 2]
```diff
@@ -7,12 +7,7 @@
     </template>
     <template v-slot:aboveExplanation>
       <p>
-        When scientists observe a large <span class="emph">snowpack magnitude</span>, a late <span class="emph">melt timing</span> 
-        and short <span class="emph">melt duration</span>, they mark that as a high snow year.  The later the peak snowpack day occurs, 
-        the more likely it is that the snow will melt all at once in a rapid surge. When these dynamics line up, it's a good 
-        estimation that there will be a good amount of <span class="emph">flow</span> to provide good water available for the 
-        rest of the year, even if the <span class="emph">soil saturation</span> soaks up some of the water. Even dry soils aren’t 
-        able to absorb all the meltwater, and there can even be spring flooding.  
+        Scientists have observed that in years with lots of water availability, the snow dynamics line up to produce a huge surge of flow during the spring melt.
```
Comment (collnell) [inline src/components/DiagramsHigh.vue:10]: This sentence needs workshopping, it's too vague. Not clear what is meant by "snow dynamics line up". Write about the process, not the observation, i.e. In years with a lot of snow, spring snowmelt generates a large pulse of flow through the environment.
The sentence is also backwards, high availability doesn't cause snow, snow is the cause of high availability.

File: src/components/SWEanim.vue [hunk 3]
```diff
@@ -139,16 +139,26 @@
       <p>
         Elevation is intertwined with numerous factors like snow persistence, wind redistribution, and slope that drive complex snow-to-flow dynamics from site to site. At higher elevations, fallen snow can be blown over ridges, scouring windward rises or trees, leading to snow accumulating on the leeward side of the ridge. This results in spatial differences in snowpack depth that contribute to variation in snowmelt timing and rates, in addition to other climatic factors.
       </p>
+      <Sidebar>
+        <template v-slot:sidebarTitle>
+          Why is the year measured from October-to-October?
+        </template>
+        <template v-slot:sidebarMessage>
+          <p>Hydrologists use a <span class="emph">water year</span> to delineate a  year because it captures a full snow cycle.</p>
```
Comment (collnell) [inline src/components/SWEanim.vue:148]: needs  more explanation https://water.usgs.gov/nwc/explain_data.html

File: src/components/SNTLmap.vue [hunk 4]
```diff
@@ -1462,6 +1462,14 @@
     </template>
     <!-- EXPLANATION -->
     <template v-slot:belowExplanation>
+      <Sidebar>
+        <template v-slot:sidebarTitle>
+          But peak SWE hasn't happened yet in some places.
```
Comment (collnell) [inline src/components/SNTLmap.vue:1468]: How about "When is peak SWE"

Comment (ellenbechtel) [reply src/components/SNTLmap.vue:1468]: 👍

File: src/components/SWEanim.vue (hunk 3 above)
Comment (collnell) [reply src/components/SWEanim.vue:148]: and what about putting this with the first line chart where it first appears? here works too, whichever isn't too crowded

Comment (ellenbechtel) [reply src/components/SWEanim.vue:148]: 👍

File: src/components/DiagramCarouselNormal.vue (hunk 1 above)
Comment (ellenbechtel) [reply src/components/DiagramCarouselNormal.vue:21]: 👍

**Review by collnell — APPROVED** [review]

**Pull request merged**